push/src/main/java/org/aerogear/mobile/push/UnifiedPushConfig.java

```diff
@@ -0,0 +1,31 @@
+package org.aerogear.mobile.push;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public final class UnifiedPushConfig {
+
+    private String alias;
+    private List<String> categories;
+
+    public String getAlias() {
+        return alias;
+    }
+
+    public void setAlias(String alias) {
+        this.alias = alias;
+    }
+
+    public List<String> getCategories() {
+        if (categories == null) {
+            return new ArrayList<>();
+        }
+        return Collections.unmodifiableList(categories);
+    }
+
+    public void setCategories(List<String> categories) {
+        Collections.copy(this.categories, categories);
```

Comment:
if `this.categories` is null or is not big enough to contain the passed in list, this will throw an exception.
Better change this to 
```java
this.categories = new ArrayList(categories)
```